app/uk/gov/hmrc/customs/declaration/xml/MdgPayloadDecorator.scala

```diff
@@ -21,10 +21,12 @@ import org.joda.time.format.ISODateTimeFormat
 import uk.gov.hmrc.customs.declaration.model._
 import uk.gov.hmrc.customs.declaration.model.actionbuilders.{AuthorisedStatusRequest, ValidatedPayloadRequest}
 
-import scala.xml.NodeSeq
+import scala.xml.{Node, NodeSeq, Text}
 
 class MdgPayloadDecorator() {
 
+  private val newLineAndIndentation = "\n        "
```

Comment:
I didn't think the change was needed here

Reply:
it was outputting some nodes on one single line

Reply:
ok